go/test/endtoend/onlineddl/vrepl/onlineddl_vrepl_test.go

```diff
@@ -171,6 +174,7 @@ func TestMain(m *testing.M) {
 			"--throttle_threshold", "1s",
 			"--heartbeat_enable",
 			"--heartbeat_interval", "250ms",
+			"--heartbeat_on_demand_duration", "5s",
```

Comment:
re-adding `--heartbeat_on_demand_duration`; the most interesting scenario in this `endtoend` suite is a PRS that runs throughout a throttled migration.